<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>932</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Interfaz Supervisión</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>doggy.png</normaloff>doggy.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 4, 75);</string>
  </property>
  <widget class="QLabel" name="label_orden_5">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>400</y>
     <width>150</width>
     <height>173</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(240, 255, 188);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>doggy.png</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_orden_6">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>289</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(240, 255, 188);</string>
   </property>
   <property name="text">
    <string>Interfaz para Supervisar</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>50</y>
     <width>401</width>
     <height>581</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="title">
    <string>Gráficas</string>
   </property>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>361</width>
      <height>551</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLabel" name="label_orden_3">
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Tensión vs Tiempo</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="PlotWidget" name="graphicsView">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_orden_4">
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(240, 255, 188);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string> Espectro en Frecuencias</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="PlotWidget" name="graphicsView_2">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>471</width>
     <height>281</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(170, 170, 255);</string>
   </property>
   <property name="title">
    <string>Estado de la Sesión</string>
   </property>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>411</width>
      <height>111</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLabel" name="label_orden">
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Comenzando...</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="styleSheet">
      <string notr="true">QProgressBar::chunk{background-color: rgb(255, 170, 0)}
border-color: rgb(0, 0, 0);
</string>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
     <property name="format">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>431</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_sesion">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Tipo de Sesión:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_trial">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Tiempo del Trial:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_etapa">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Etapa Actual:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_canal">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Canales Seleccionados:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
